README.md

```diff
@@ -15,3 +15,74 @@ Accessibility
 
 Evaluation Date: 201X-XX-XX  
 This project conforms to level AA WCAG 2.0 standards as required by the university's accessibility policy. For more information on the policy please visit: [https://ucomm.stanford.edu/policies/accessibility-policy.html](https://ucomm.stanford.edu/policies/accessibility-policy.html).
+
+System Requirements
+---
+
+To build and deploy code, you must have the following installed on your system:
+
+* Node
```

Comment:
Perhaps also add version numbers?

Reply:
+1. The more documentation (and the more specific), the better here, I think. Let's be diligent about making sure we keep this updated as things change.